src/system.rs

```diff
@@ -154,3 +156,61 @@ pub fn sysinfo() -> Sysinfo {
 pub fn sethostname(name: &[u8]) -> io::Result<()> {
     backend::system::syscalls::sethostname(name)
 }
+
+/// Reboot command to be used with [`reboot`]
+///
+/// [`reboot`]: crate::system::reboot
+#[cfg(target_os = "linux")]
+#[derive(Copy, Clone, Debug, Eq, PartialEq)]
+#[repr(i32)]
+#[non_exhaustive]
+pub enum RebootCommand {
+    /// CAD is disabled.
+    /// This means that the CAD keystroke will cause a SIGINT signal to be sent to init (process 1),
+    /// whereupon this process may decide upon a proper action (maybe: kill all processes, sync, reboot).
```

Comment:
My general goal for rustix is that we should either write documentation ourself, or link to Linux's documentation, but not copy and paste whole paragraphs from Linux's documentation. If we're copying and pasting, developers would often be better off referring to Linux's documentation, which often has more context. In this case, Linux's documentation includes context like what "CAD" stands for, and the fact that the key sequence can be changed.

I know that having docs in Rust comments is more convenient for Rust developers. On the other hand, this is `reboot` and it's not something anyone is going to be using on a regular basis, and it has some subtle behavior not included here, so it really would be better to encourage developers to read the Linux docs directly, rather than copying and pasting parts of them here.

Reply:
What would you recommend we do then? I copied and pasted it because I felt like it made sense but I can see it is missing a bunch of context. Maybe we can just not document it and refer user to the man page, something like reboot should probably not be touched by someone who hasn't read it.

Reply:
Yeah, I'd say leave most of the text out, perhaps just add short one-line descriptions if anything, and add a link to the Linux docs in `RebootCommand`'s doc comment to help people find it.